back/.air.toml

```diff
@@ -3,8 +3,7 @@ testdata_dir = "testdata"
 tmp_dir = "tmp"
 
 [build]
-  args_bin = ["serve"]
-  bin = "bin/server"
+  bin = "/bin/bash -c 'source refresh-env && bin/server serve'"
```

Comment:
why hardcoding the command in the bin variable instead of using args_bin has before ?

Reply:
I had issue making it work with the env refresh, this allows you to refresh the environment while keeping the api open (and hot reload it)